code.ts

```diff
@@ -1,11 +1,15 @@
 const delay = ms => new Promise(res => setTimeout(res, ms));
-var activeTimer = 0;
+var totalTimers = 0;
 const secondsSet = [86400, 3600, 60, 1];
 var pause = false;
 var reset = false;
 var userSetSeconds = 0;
 
-figma.showUI(__html__, { width: 220, height: 50 })
+var uiHeight = 60;
+var timerUIHeight = 50;
+var uiMaxHeight = 300;
```

Comment:
all of these vars contain UI in there name. Maybe it makes sense to refactor it into an Object which could be called UIConfiguration

Reply:
Like this? @lennet 

`var uiWindow = {
  minHeight: 60,
  maxHeight: 300,
  helptextHeight: 200,
  width: 220,
}`

Reply:
Yes! You can even make it const